atlassian/confluence.py

```diff
@@ -2798,17 +2800,18 @@ def get_atl_request(url: str):
             parsed_html = BeautifulSoup(response.text, "html.parser")
             # Getting the poll URL to get the export progress status
             try:
-                poll_url = parsed_html.find("meta", {"name": "ajs-pollURI"}).get("content")
+                poll_url = cast("str", parsed_html.find("meta", {"name": "ajs-pollURI"}).get("content"))  # type: ignore[union-attr]
             except Exception as e:
                 raise ApiError("Problems with getting the poll_url for get_space_export method :", reason=e)
             running_task = True
             while running_task:
                 try:
                     progress_response = self.get(poll_url)
+                    assert progress_response
```

Comment:
Mypy see `get` could return `None`.  Since the following code appears to assume it wasn't `None`, asserting here made the most sense.